### PR TITLE
[Warhammer 4e Character Sheet] v1.74b fixes & updates

### PR DESCRIPTION
**Pull request by Djjus**
- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

# Changes / Description

Dec 14th 2024 v1.74b

- New Custom Race added to Race dropdown, will allow entering own Race. See setting's tab for Custom Race Stating Characteristics Modifier, this allows the Char Roll to generate starting Charactistics for the Custom Race.
- Fix issue with Dual Wield trigger checkbox not showing correctly in Declare Actions section.
- Fix issue with impact/damaging not showing on Attack rolls with Size modifier Large Vs Small.
- Fix mixed up Impact/Impale checkboxes on the NPC pages.
- Fix Dangerous Quality fumble sometimes not showing when test Unit or 10s Die is 9.
- Fix Warpstone Quality now only triggered on crit again.
- Resolved some issues with dulpicate crit translations.
- Cleaned up minor CSS file errors.

### CODE REVIEW

File: Warhammer 4e Character Sheet/Warhammer 4e Character Sheet.html [hunk 1]
```diff
@@ -27634,25 +27899,25 @@
...
-									<option data-i18n="AVERAGE" selected>Average</option>
-									<option data-i18n="LONG">Long</option>
-									<option data-i18n="VERY-LONG">Very Long</option>
-									<option data-i18n="MASSIVE">Massive</option>
+									<option value="Personal" data-i18n="PERSONAL">Personal</option>
+									<option value="Very Short" data-i18n="VERY-SHORT">Very Short</option>
+									<option value="Short" data-i18n="SHORT">Short</option>
+									<option value="Average" data-i18n="AVERAGE" selected>Average</option>
+									<option value="Long" data-i18n="LONG">Long</option>
+									<option value="Very Lon" data-i18n="VERY-LONG">Very Long</option>
```
Comment (NorWhal) [inline Warhammer 4e Character Sheet/Warhammer 4e Character Sheet.html:27919]: typo on this value here

File: Warhammer 4e Character Sheet/translation.json [hunk 2]
```diff
@@ -1893,7 +1886,128 @@
...
+	"WEAPON-STAFF-SLING": "2H,Staff Sling,Sling,2,100,0,7,none,",
+
+	"WEAPON-BOLAS": "1H,Bolas,Throwing,0,SB3,1,0,Entangle,",
+	"WEAPON-DART": "1H,Dart,Throwing,0,SB2,1,1,Impale,",
+	"WEAPON-JAVELIN": "1H,Javelin,Throwing,1,SB3,1,3,Impale,",
+	"WEAPON-ROCK": "1H,Rock,Throwing,0,SB1,1,0,none,",
+	"WEAPON-THROWING-KNIFE": "1H,Throwing Knife,Throwing,0,SB2,1,2,none,",
+	"WEAPON-THROWING-AXE": "1H,Throwing Axe,Throwing,1,SB2,1,3,Hack,",
+	"WEAPON-DWARF-THROWING-AXE": "1H,Dwarf Throwing Axe,Throwing,1,SB2,1,4,Hack,",
+	"WEAPON-BLACKBRIAR-JAVELIN": "1H,Blackbriar Javelin,Throwing,1,SB3,1,3,Impale,Inflicts 1 Poison on hit",
```
Comment (NorWhal) [inline Warhammer 4e Character Sheet/translation.json:2010]: These can't be included. Community sheets cannot include content from the books. This aspect will need to be removed before this PR can be approved.

Comment (Djjus) [reply Warhammer 4e Character Sheet/translation.json]: @NorWhal I have removed the offending PR related content. Need me to open a new request?

**Comment by Djjus** [comment]
@NorWhal I have removed the offending PR related content. Need me to open a new request?

**Pull request merged**